static/app/views/organizationGroupDetails/groupDetails.tsx

```diff
@@ -176,6 +176,7 @@ class GroupDetails extends Component<Props, State> {
       issue_level: group?.level,
       is_assigned: !!group?.assignedTo,
       error_count: Number(group?.count || -1),
+      error_has_replay: Boolean(event?.tags?.find(({key}) => key === 'replayId')),
```

Comment:
Double checking here, are we interested in just the current event in the issue or do we want to know if any event in the issue has a replay?